expo/gbe/models.py

```diff
@@ -1588,6 +1603,86 @@ def __unicode__(self):
         return self.bidder.display_name
 
 
+class Costume(Biddable):
+    '''
+    An offer to display a costume at the Expo's costume display
+      - profile is required, persona is optional
+      - debut date is a text string to allow vague descriptions
+      - "creator" is put into Biddable title as both are required
```

Comment:
No. If we are recording "creator" we're not going to put it in a field called "title".